src/test/resources/test-without-stop-task.yml

```diff
@@ -0,0 +1,10 @@
+server:
+  applicationConnectors:
+    - type: http
+      port: 0
+  adminConnectors:
+    - type: http
+      port: 0
+sundial:
+  thread-pool-size: 1
+  tasks: [startjob]
```

Comment:
No, it lists which tasks are enabled (this test only enables startjob), the default is all tasks enabled.